src/OMSimulatorLib/TLM/TLMCompositeModel.cpp

```diff
@@ -294,6 +301,36 @@ oms_status_enu_t oms2::TLMCompositeModel::addConnection(const SignalRef &signalA
   return addConnection(oms2::TLMConnection(this->getName(),signalA,signalB,delay,alpha,Zf,Zfr));
 }
 
+bool oms2::TLMCompositeModel::exists(const oms2::ComRef &cref)
+{
+  if(!cref.isIdent()) {
+    FMICompositeModel *fmiModel = getFMIModel(cref.first());
+    if(NULL != fmiModel) {
+      FMISubModel *fmu = fmiModel->getSubModel(cref.last(), false);
+      return (NULL != fmu);
```

Comment:
I think there is one more case missing: `tlmModel.fmiModel.solver`

Reply:
That's true. I added a check for that as well.